x-pack/plugins/infra/public/components/asset_details/tabs/overview/metadata_summary.tsx

```diff
@@ -42,6 +46,8 @@ const metadataData = (metadataInfo: InfraMetadata['info']) => [
   {
     field: 'hostIp',
     value: metadataInfo?.host?.ip,
+    tooltipLinkLabel: 'host.ip',
+    tooltipLink: 'https://www.elastic.co/guide/en/ecs/current/ecs-host.html#field-host-ip',
```

Comment:
For now, I think it's ok, but once we start integrating more asset types, we might have to rely on what `findInventoryModel(nodeType)` returns. `ip` info might sit in a different field name depending on the asset type. The same goes for the translations.

Reply:
Good point, we should revisit this once we have more asset types 👍